micro-ui/web/micro-ui-internals/packages/modules/pgr/src/configs/UICustomizations.js

```
@@ -0,0 +1,281 @@
+    } else {
+      return inboxModuleNameMap;
+    }
+  },
+
+  PGRInboxConfig: {
+    preProcess: (data) => {
+      data.body.inbox.tenantId = Digit.ULBService.getCurrentTenantId();
+      data.body.inbox.processSearchCriteria.tenantId = Digit.ULBService.getCurrentTenantId().split(".")[0];
+      //data.body.inbox.moduleSearchCriteria.tenantId = Digit.ULBService.getCurrentTenantId();
+
+      if(data?.body?.inbox?.moduleSearchCriteria?.mobileNumber) delete data.body.inbox.moduleSearchCriteria.mobileNumber
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)